.gitignore

```diff
@@ -1,5 +1,19 @@
+#test junk
 tests/data/basic_playbook.retry
 tests/data/basic_playbook_with_volume.retry
 contrib/ab-pod.yml
 .coverage.*
 .coverage*
+
+#python junk
```

Comment:
can well call this with a different name than "junk", sounds a little offensive

how about "test artifacts" and "python artifacts"?